RogueSharp/Algorithms/DepthFirstPaths.cs

```diff
@@ -60,14 +60,15 @@ public bool HasPathTo( int destinationVertex )
       /// Returns a path between the sourceVertex and the specified destinationVertex or null if no such path exists
       /// </summary>
       /// <param name="destinationVertex">The destination vertex</param>
-      /// <returns>An IEnumerable sequence of vertices representing the path between the sourceVertex and the specified destinationVertex or null if no such path exists</returns>
+      /// <returns>An IEnumerable sequence of vertices representing the path between the sourceVertex and the specified destinationVertex or null if no such path exists. The path will not include the source vertex</returns>
       public IEnumerable<int> PathTo( int destinationVertex )
       {
          if ( !HasPathTo( destinationVertex ) )
          {
             return null;
          }
          var path = new Stack<int>();
+         // TODO: All methods that return paths should be consistent on whether or not they include the source and destination vertices in the path.
```

Comment:
Do you really want to leave a TODO in code? It might be better to track this as an issue.

Reply:
I didn't want to make major changes which would cause braking changes. Do agree that they could be issues though instead. I was being lazy.